CHANGELOG.rst

```diff
@@ -2,6 +2,66 @@
 SDK Changelog
 =============
 
+1.5.0 (2023-08-23)
+==================
+
+New SDK minor version.
+
+We've worked on a couple of big topics in this release:
+
+* a tailored flow for appointment forms. Legacy appointments features are now
+  deprecated and will be removed in SDK 2.0.
+* improved handling of maps/geographical information.
+* various improvements for NL Design System integration, which is still an ongoing effort.
+
+.. warning:: SDK 1.5.0-alpha.0 requires at least version 2.3.0 of the Open Formulieren API.
```

Comment:
Maybe this should say 1.5.0 and 1.5.0-alpha.0 (or maybe not mention the alpha at all?)

Reply:
oh nice catch, bad copy paste! I change it to `1.5.0`